tests/cucumber/e2e/stepDefinitions/loginContext.js

```diff
@@ -0,0 +1,26 @@
+// A new context file
+const { Given, When, Then } = require ("@cucumber/cucumber")
+const { expect } = require("@playwright/test")
+const { LoginPage } = require('../pageObjects/LoginPage')
+const loginPage = new LoginPage
+
+Given('the user has browsed to the login page', async function () {
+  expect(await loginPage.navigateLogin()).toBeVisible()
+  // await expect(page.locator(loginPage.homepageSelector)).toBeVisible()
+});
```

Comment:
First write the code that navigates you to the login page, then check if you have navigated to the login page.
```suggestion
Given('the user has browsed to the login page', async function () {
  await loginPage.navigateLogin()
  await expect(page.locator(loginPage.loginButtonSelector)).toBeVisible()
});
```